src/pubsub/ua_pubsub.h

```diff
@@ -146,7 +146,8 @@ UA_DataSetWriter_findDSWbyId(UA_Server *server, UA_NodeId identifier);
 
 UA_StatusCode
 UA_DataSetWriter_setPubSubState(UA_Server *server, UA_PubSubState state,
-                                UA_DataSetWriter *dataSetWriter);
```

Comment:
Just a small nitpick.
Since we are touching the api, I prefer the following order of the arguments:

server -> writer -> state -> cause

first the context, then the change.

Reply:
Order changed.